backend.native/tests/framework/values/values.kt

```diff
@@ -935,4 +936,43 @@ class TestStringConversion {
     lateinit var str: Any
 }
 
-fun foo(a: kotlin.native.concurrent.AtomicReference<*>) {}
\ No newline at end of file
+fun foo(a: kotlin.native.concurrent.AtomicReference<*>) {}
+
+interface GH3825 {
```

Comment:
This values.kt gets awfully big. Cannot we split it, or at least add new tests to separate files?

Reply:
Indeed.
Actually I am going to improve this test infrastructure and start adding tests to separate files with next PR.